packages/react/src/primitives/shared/constants.ts

```diff
@@ -72,6 +72,14 @@ export enum ComponentClassNames {
   SwitchThumb = 'amplify-switch-thumb',
   SwitchTrack = 'amplify-switch-track',
   SwitchWrapper = 'amplify-switch__wrapper',
+  Table = 'amplify-table',
+  TableBody = 'amplify-table__body',
```

Comment:
I think there should also be TableCaption classname:
```suggestion
  TableBody = 'amplify-table__body',
  TableCaption = 'amplify-table__caption',
```

Reply:
Maybe a `TableCaption` component? But would like to see how you are about to structure the table